### ISSUE TITLE
Wrong typescript exports

### ISSUE THREAD

**Issue by vaiil** [vue-final/vue-final-modal#228]
### Version

vue-final-modal: 3.4.4
vue: 3.2.33

### OS

Linux

### Reproduction Link
https://codesandbox.io/s/icy-pine-fm35pi?file=/src/App.vue

### Steps to reproduce
Try to import:
```ts
import { VueFinalModal } from 'vue-final-modal'; 
```
### What is Expected?
It should be imported without TS warnings.

### What is actually happening?
TS shows error, that this module not found
![image](https://user-images.githubusercontent.com/3755615/169063820-fc6727a5-5681-4477-b863-384cdb06b316.png)

Default import doesn't work at all.

**Comment by HabardiT** [comment]
I can confirm this bug exists

> I can confirm this bug. Try importing single items without braces.
> 
> **Solution** `import VueFinalModal from 'vue-final-modal'`

I tried it, it doesn't work unfortunately, it shows the below warning and the modal doesn't work.

```
[Vue warn]: Invalid VNode type: undefined (undefined) 
  at <Ze modelValue=false onUpdate:modelValue=fn > 
  at <Modal> 
  at <App>
```

**Comment by carlfranciscastillo** [comment]
Any updates on this? Seems weird that its flagged as an error when it really isn't

**Comment by alexpvieira** [comment]
@hunterliu1003 the PR that I created should fix this problem. It's possible that you could review it?

**Comment by hunterliu1003** [comment]
Released in [v3.4.6](https://www.npmjs.com/package/vue-final-modal/v/3.4.6)
